src/emsesp.cpp

```diff
@@ -386,13 +386,19 @@ void EMSESP::show_device_values(uuid::console::Shell & shell) {
 
                 // print line
                 for (JsonPair p : json) {
-                    const char * key = p.key().c_str();
-                    shell.printf("  %s: ", key);
+                    std::string key = p.key().c_str(); // this will be fullname and the shortname in brackets
+                    shell.printf("  %s: ", key.c_str());
                     JsonVariant data = p.value();
                     shell.print(COLOR_BRIGHT_GREEN);
                     shell.print(data.as<std::string>());
+
+                    // extract the shortname from the key, which is in brackets
+                    std::size_t first_bracket = key.find_first_of('(');
```

Comment:
We have some long names with brackets lke `temperature (TS1)` or `valve (VS2)`, better check for last open/close brackets.

Reply:
Thanks for reviewing. I'll see if I can scan backwards for the last bracket as you suggested

Reply:
fixed.